certblob/generate.sh

```diff
@@ -4,16 +4,16 @@ set -euo pipefail
 shopt -s nullglob globstar
 
 # wincrypt.h was downloaded from the following URL:
-# https://raw.githubusercontent.com/reactos/reactos/8ae8083378546fc7d907e489aeebddbe3c8d9399/sdk/include/psdk/wincrypt.h
-# Note that wincrypt.h only has property ID's from Windows XP.
-# certenroll.h appears to have more (from Windows Vista) but it's not in ReactOS.
+# https://github.com/mingw-w64/mingw-w64/blob/master/mingw-w64-headers/include/wincrypt.h
+# wincrypt.h is implemented from  MinGW-w64 and has property ID's upto 127.
+
```

Comment:
Please mention that `certenroll.h` is an alternative that exists in MinGW, but that MinGW's `certenroll.h` is missing many property ID's that exist in MinGW's `wincrypt.h`.

Reply:
Also please mention that we use MinGW's header instead of ReactOS's because MinGW has more property ID's.

Reply:
Implemented